src/Uno.UWP/Devices/Sensors/Magnetometer.wasm.cs

```diff
@@ -0,0 +1,78 @@
+﻿#if __WASM__
+using System;
+using System.Diagnostics;
+using Uno;
+using Uno.Devices.Sensors.Helpers;
+
+namespace Windows.Devices.Sensors
+{
+	public partial class Magnetometer
+	{
+		private const string JsType = "Windows.Devices.Sensors.MagnetometerSensor";		
```

Comment:
Just a question, we cannot match exactly the same namespace "Windows.Devices.Sensors.Magnetometer" ?

Reply:
Great question, I would need some help here as I hardly know any Typescript syntax :-D. The problem is browser has a global API called Magnetometer and I don't know how to reference it while avoiding nameclash with the class itself 🤔

Reply:
Now matches the namespace. Using the window. global object to access Magnetometer